### PR TITLE
Add new UI for dust app run details

### PR DESCRIPTION
**Pull request by flvndvd**
## Description

This PR applies the same design as in https://github.com/dust-tt/dust/pull/5500 to dust app run.

image

### CODE REVIEW

File: front/components/actions/dust_app_run/DustAppActionDetails.tsx [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+        style={{
+          "hljs-number": {
+            color: amber["500"],
+          },
+          "hljs-literal": {
+            color: amber["500"],
+          },
+          "hljs-string": {
+            color: emerald["600"],
+            // @ts-expect-error - this is a valid style
```
Comment (fontanierh) [inline front/components/actions/dust_app_run/DustAppActionDetails.tsx:92]: weird

**Review by fontanierh — APPROVED** [review]
Nice !

Since you are here, any chance we could get a copy to clipboard button ?

**Comment by flvndvd** [comment]
> Nice !
> 
> Since you are here, any chance we could get a copy to clipboard button ?

Yep coding in a follow up PR, I extracted the code block from the react message rendering.

File: front/components/actions/dust_app_run/DustAppActionDetails.tsx (hunk 1 above)
Comment (flvndvd) [reply front/components/actions/dust_app_run/DustAppActionDetails.tsx:92]: Cargo culted 🙈

**Pull request merged**